wire-ios-data-model/Tests/Source/Model/Observer/SearchUserObserverCenterTests.swift

```diff
@@ -18,184 +18,13 @@
 
 @testable import WireDataModel
 
-class SearchUserSnapshotTests: ZMBaseManagedObjectTest {
```

Comment:
There were two classes in one file, so I've moved it to a new file `SearchUserSnapshotTests.swift`